conf/messages

```diff
@@ -228,7 +228,7 @@ tracing.wrongMag=Tracing “{0}” does not have mag “{1}”
 
 annotation.create.failed=Failed to create annotation.
 annotation.create.forbidden=You do not have permission to create annotations for this dataset.
-annotation.volume.negativeLargestSegmentId=The resulting volume annotation would have an invalid largest segment id (must be nonnegative, within element class range, and < 2^53)
+annotation.volume.invalidLargestSegmentId=Cannot create tasks with fallback segmentation layers that do not have a valid largest segment id.
```

Comment:
@fm3  This is only a theoretical warning, right? e.g, if we would allow to create volume tasks by specifying a layer, this would come up (not possible currently).
Just asking, because I just checked volume tasks by specifying an annotation ID or a zip as a base and it worked.

Reply:
Yes, I think this is not currently possible. But some of the code paths already exist and I wanted to make sure they’re guarded

Reply:
ok, great :+1: